sentinel-dashboard/src/main/resources/application.properties

```diff
@@ -3,6 +3,9 @@ spring.http.encoding.force=true
 spring.http.encoding.charset=UTF-8
 spring.http.encoding.enabled=true
 
+#cookie name setting
+server.servlet.session.cookie.name=${SENTINEL_COOKIE_NAME:sentinel}
+
```

Comment:
Using placeholders, the default value is sentinel, and we can change it 
by VM argument `-DSENTINEL_COOKIE_NAME=xxx`. 
Now the argument name `SENTINEL_COOKIE_NAME` is uppercase, since others such as`auth.username` and `auth.password`.. are lower case with dot separate, they are not unified.

It seems that only use `sentinel` or `sentinel_cookie`、`sentinel_dashboard_cookie` is enough, since we can change it by `-Dserver.servlet.session.cookie.name=xxx`.

Reply:
Above are some suggestions, sorry for choosing dyslexia myself, and I think ${SENTINEL_COOKIE_NAME:sentinel} is also acceptable.
How about your opinion? @zhaoyuguang @sczyh30

Reply:
仔细想下 `${SENTINEL_COOKIE_NAME:`的确是多余的，把默认的cookie key改成了`sentinel_dashboard_cookie`这个看起来比较清晰 @cdfive